dashboard/package.json

```diff
@@ -56,13 +56,13 @@
     "prettier": "prettier --write 'src/**/*.{ts,tsx,css}'",
     "lint-js": "tslint -c ./tslint.ci.json 'src/**/*.{ts,tsx}'",
     "lint-js-fix": "tslint -c ./tslint.ci.json --fix 'src/**/*.{ts,tsx}'",
-    "lint-css": "sass-lint -c ./.sass-lint.yml -v",
-    "lint-css-fix": "sass-lint-auto-fix -c ./.sass-lint.yml",
+    "lint-css": "stylelint --config ./.stylelintrc.json 'src/**/*.scss'",
+    "lint-css-fix": "stylelint --config ./.stylelintrc.json --fix 'src/**/*.scss'",
     "lint": "npm-run-all lint-js lint-css"
   },
   "lint-staged": {
     "*.{ts,tsx}": ["yarn run lint-js-fix", "prettier --write", "git add"],
-    "*.scss": ["yarn run lint-css-fix", "yarn run lint-css", "prettier --write", "git add"],
+    "*.scss": ["yarn run lint-css-fix", "prettier --write", "git add"],
```

Comment:
so lint-css is not needed anymore? I thought that we wanted to add it as a guard before git add?

Reply:
sass-lint didn't have autofix so there was a separate tool for that, stylelint has it built in and will do the lint and fixing in one go, similar to tslint.

r.e. running prettier before linting, I think we should do that in a separate PR, and I'm still not 100% sure we should do that. I'd like to understand more about why it was in this order in the first place.